docs/apache-airflow/administration-and-deployment/logging-monitoring/logging-tasks.rst

```diff
@@ -57,34 +57,30 @@ In addition, you can supply a remote location to store current logs and backups.
 Writing to task logs from your code
 -----------------------------------
 
+Airflow uses standard Python `logging <https://docs.python.org/3/library/logging.html>`_ framework to
+write logs, and for duration of task, root logger is configured to write to task log.
+
 Most operators will write logs to the task log automatically. This is because they derive from they
 have a ``log`` logger that you can use to write to the task log.
-This logger is created and configured by :class:`~airflow.utils.log.LoggingMixin` that all classic
-operators derive from.
-
-If you want to log to the task log from a custom class of yours you can do the following:
+This logger is created and configured by :class:`~airflow.utils.log.LoggingMixin` that all
+operators derive from. But also due the root logger handling, any standard logger that (default setting)
+propagates logging to the root will also write to the task log.
 
-* make sure your class extends from :class:`~airflow.utils.log.LoggingMixin`
-* just use standard print statements to print to stdout
-* use the ``airflow.task`` logger that is configured by default or create logger where ``airflow.task`` is
-  the parent logger
+So if you want to log to the task log from a custom code of yours you can do the following:
 
-The last option is the most flexible, as you can create your own logger and configure it as you see fit
-via advanced configuration options below.
+* make sure your class extends from :class:`~airflow.utils.log.LoggingMixin` and use ``self.log`` logger
+* just use standard print statements to print to stdout (not recommended, but in some cases it can be useful)
+* use the standard logger approach of creating a logger using python module name
+  and using it to write to the task log
```

Comment:
```suggestion
* Make sure your class extends from :class:`~airflow.utils.log.LoggingMixin` and use the ``self.log`` logger
* Just use standard ``print`` statements to print to ``stdout`` (not recommended, but in some cases it can be useful)
* Use the standard logger approach of creating a logger using the Python module name
  and using it to write to the task log
```